map2loop/project.py

```diff
@@ -446,18 +430,54 @@ def calculate_stratigraphic_order(self, take_best=False):
                 self.map_data.contacts,
                 self.map_data,
             )
-
+    
+    
     def calculate_unit_thicknesses(self):
         """
         Use the stratigraphic column, and fault and contact data to estimate unit thicknesses
         """
-        self.stratigraphic_column.stratigraphicUnits = self.thickness_calculator.compute(
-            self.stratigraphic_column.stratigraphicUnits,
-            self.stratigraphic_column.column,
-            self.map_data.basal_contacts,
-            self.structure_samples,
-            self.map_data,
-        )
+        thickness_calculator_labels = ['ThicknessCalculatorAlpha', 'InterpolatedStructure', 'StructuralPoint', ]
+
+        thickness_calculation_results = []
+        
+        for label in thickness_calculator_labels:
+            if label == 'InterpolatedStructure':
+                calculator = InterpolatedStructure()
+            elif label == 'StructuralPoint':
+                calculator = StructuralPoint()
+            elif label == 'ThicknessCalculatorAlpha':
+                calculator = ThicknessCalculatorAlpha()
+                
+            result = calculator.compute(
+                self.stratigraphic_column.stratigraphicUnits,
+                self.stratigraphic_column.column,
+                self.map_data.basal_contacts,
+                self.structure_samples,
+                self.map_data,
+            )[['ThicknessMean', 'ThicknessMedian', 'ThicknessStdDev']]
+            
+            # Rename the columns
+            result = result.rename(columns={
+                'ThicknessMean': f'ThicknessMean_{label}',
+                'ThicknessMedian': f'ThicknessMedian_{label}',
+                'ThicknessStdDev': f'ThicknessStdDev_{label}'
+            })
+            
+            thickness_calculation_results.append(result)
+            
+        thickness_combined_results = pandas.concat(thickness_calculation_results, axis=1)
+
+        self.stratigraphic_column.stratigraphicUnits.loc[:, 'ThicknessMean_ThicknessCalculatorAlpha'] = thickness_combined_results['ThicknessMean_ThicknessCalculatorAlpha']
```

Comment:
Could you not just put the thickness_combined_results directly inside the stratigraphicUnits inside the for loop. This means you aren't hard coding the number of thickness calculators